README.md

```diff
@@ -144,6 +145,38 @@ $ spanner-cli -p myproject -i myinstance -d mydb -e 'SELECT * FROM users;' -t
 +----+------+--------+
 ```
 
+### Directed reads mode
+
+spanner-cli now supports directed reads, a feature that allows you to read data from a specific replica of a Spanner database. 
+To use directed reads with spanner-cli, you need to specify the `--directed-read` flag.
+The `--directed-read` flag takes a single argument, which is the name of the replica that you want to read from.
+The replica name can be specified in one of the following formats:
+
+- `<replica_location>`
+- `<replica_location>:<replica_type>`
+
+The `<replica_location>` part of the replica name specifies the region where the replica is located such as `us-central1`, `asia-northeast2`.  
+The `<replica_type>` part of the replica name specifies the type of the replica either `READ_WRITE` or `READ_ONLY`. 
```

Comment:
Maybe "The `<replica_type>` specifies the type of ..." would be simpler?

Reply:
fix